src/shellhistory/shellhistory.sh

```diff
@@ -125,23 +139,27 @@ _shellhistory_after() {
 }
 
 _shellhistory_enable() {
-  # mkdir -p "${SHELLHISTORY_ROOT}" &>/dev/null
+  _SHELLHISTORY_BEFORE_DONE=2
+  _SHELLHISTORY_AFTER_DONE=1
   if [ "${ZSH_VERSION}" ]; then
-    _shellhistory_command_type() { _shellhistory_zsh_command_type "$1"; }
       # FIXME: don't override possible previous contents of precmd
```

Comment:
This needs to be fixed before merging.

Reply:
Fixed in https://github.com/pawamoy/shell-history/pull/23/commits/7336a8c347fb0d593b74e3b501f51cf484eb4afd